### ISSUE TITLE
Build Session - series 'shuffle' play (suggestion)

### ISSUE THREAD

**Issue by chrismorel** [GShotwell/shambhala_app_issues#41]
When building a session, the user can currently select a talk or the entire series.  Suggestion - As an alternative to selecting the series (all talks), allow the user to select the series as a "shuffle" play.   (Though, still allow the user to also select all talks from a series.)  In this way, when the session is played, each time it would play a different talk from the series (either choosing randomly or rotating through the series' talks)

Is this something that users would find useful? (I'm not sure, but wanted to record this suggestion  :-)
Another consideration is how difficult would this be to implement technically.

**Comment by gshotwell** [comment]
Yeah I think it's not a bad idea and wouldn't be hard to implement. The tricky question would be how to communicate the option to the user. So maybe let's park this is Someday and think about how we could communicate this clearly in the UI.

**Comment by chrismorel** [comment]
Good point about the need for clear UI.
Glad to hear that it is easy to do technically.  This would encourage re-use of the feature, since the talk would vary.
